LINK/usr/bin/evm_watchdog.rb

```diff
@@ -2,7 +2,8 @@
 # description: ManageIQ watchdog application loop
 #
 
-require '/var/www/miq/vmdb/gems/pending/util/system/evm_watchdog'
+require 'manageiq-gems-pending'
+require 'system/evm_watchdog'
```

Comment:
Wow, yes!  Wait, but what puts util on the load path?  I believe the first require just ends up going [here](https://github.com/ManageIQ/manageiq-gems-pending/blob/4b7fdb099632dc1b2a9aefc90db149c04f52a97c/lib/manageiq/gems/pending.rb).  How does it find system?